packages/singleton-manager/demo/singleton-complex/node_modules/overlays/instance.js

```diff
@@ -0,0 +1,4 @@
+import { OverlaysManager } from './index.js';
+import { singletonManager } from '../../../../index.js';
+
+export const overlays = singletonManager.get('overlays/index.js::1.x') || new OverlaysManager();
```

Comment:
Would it be useful for features to add a default value in the singletonManager for the singletons they provide?

ie.
```javascript
singletonManager.set('overlays::index.js::1.x', new OverlaysManager());
export const overlays = singletonManager.get('overlays::index.js::1.x');
```

Reply:
no as these are app level overrides... e.g. it can only be set "once" (on app level) any additional settings to the same "key" will throw an error